.circleci/config.yml

```diff
@@ -91,4 +90,4 @@ workflows:
             - bandit
           matrix:
             parameters:
-              python-version: ["3.8", "3.9", "3.10", "3.11"]
+              python-version: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
Python 3.8 is reaching its EOL in a couple of months (Oct 2024), so we may as well drop it now.